pages/en/deploying/deploying-a-subgraph-to-hosted.mdx

```diff
@@ -2,49 +2,11 @@
 title: Deploying a Subgraph to the Hosted Service
 ---
 
-To see instructions on how to deploy your subgraph to the Hosted Service, [click here](#create-a-hosted-service-account).
-
-If you have not checked out already, check out how to write the files that make up a [subgraph manifest](/developing/creating-a-subgraph#the-subgraph-manifest) and how to install the [Graph CLI](https://github.com/graphprotocol/graph-cli) to generate code for your subgraph. Now, it's time to deploy your subgraph to the Hosted Service.
-
-## Supported networks
-
-The following networks are fully supported on the Hosted Service:
-
-- `mainnet` (Ethereum)
-- `gnosis`
-
-The following networks are supported in beta on the Hosted Service:
-
-- `goerli`
-- `poa-core`
-- `poa-sokol`
-- `matic` (now known as Polygon)
-- `mumbai`
-- `fantom`
-- `bsc` (now known as BNB Chain)
-- `chapel`
-- `clover`
-- `avalanche`
-- `fuji`
-- `celo`
-- `celo-alfajores`
-- `fuse`
-- `moonriver`
-- `mbase`
-- `arbitrum-one`
-- `arbitrum-rinkeby`
-- `arbitrum-goerli`
-- `optimism`
-- `optimism-goerli`
-- `aurora`
-- `aurora-testnet`
-- `boba`
-- `harmony`
-- `zkSync2-testnet`
-
-> Ropsten, Rinkeby and Kovan are being deprecated, and are no longer supported by the Hosted Service. Read more on the [Ethereum Foundation Blog](https://blog.ethereum.org/2022/06/21/testnet-deprecation). Goerli will be maintained by client developers post-merge, and is also supported by the Hosted Service. Developers who currently use Ropsten, Rinkeby or Kovan as their staging/testing environment are encouraged to migrate to Goerli.
-
-> Subgraphs indexing Gnosis Chain can now be deployed with the `gnosis` network identifier. `xdai` is still supported for existing hosted service subgraphs.
+This page explains how to deploy a subgraph to the Hosted Service. To deploy a subgraph you need to first install the [Graph CLI](https://github.com/graphprotocol/graph-cli). If you have not created a subgraph already, see [creating a subgraph](/developing/creating-a-subgraph). 
+
+Ropsten, Rinkeby and Kovan are being deprecated, and are no longer supported by the Hosted Service. Read more on the [Ethereum Foundation Blog](https://blog.ethereum.org/2022/06/21/testnet-deprecation). Goerli will be maintained by client developers post-merge, and is also supported by the Hosted Service. Developers who currently use Ropsten, Rinkeby or Kovan as their staging/testing environment are encouraged to migrate to Goerli.
+
+Subgraphs indexing Gnosis Chain can now be deployed with the `gnosis` network identifier. `xdai` is still supported for existing hosted service subgraphs.
 
 > If an EVM compatible chain is not supported on the Hosted Service, you can run your own [graph-node](https://github.com/graphprotocol/graph-node) to index it.
 
```

Comment:
This could probably be left in both the `Deploying a subgraph to the Hosted Service` AND copied to `Supported Chains`